openai-kotlin-core/src/commonMain/kotlin/org/arvarik/openai/core/api/finetunes/CreateFineTuneResponse.kt

```diff
@@ -1,90 +1,9 @@
 package org.arvarik.openai.core.api.finetunes
 
-import kotlinx.serialization.SerialName
-import kotlinx.serialization.Serializable
-import org.arvarik.openai.core.api.EventLog
-import org.arvarik.openai.core.api.OpenAIResponse
-import org.arvarik.openai.core.api.files.File
-
 /**
  * The response object for create fine-tune API.
  *
  * @see [Fine-Tune Create API](https://beta.openai.com/docs/api-reference/fine-tunes/create)
  * @see [See the fine-tuning guide for more details](https://beta.openai.com/docs/guides/fine-tuning/creating-training-data).
  */
-@Serializable
```

Comment:
Hmm I think it's fine for now, but ideally wouldn't like to abstract common fields on Response/Request objects since it is not flexible for the future